saleor/graphql/shipping/mutations.py

```diff
@@ -11,11 +11,29 @@
 
 class ShippingPriceInput(graphene.InputObjectType):
     name = graphene.String(description="Name of the shipping method.")
-    price = Decimal(description="Shipping price of the shipping method.")
+    price = Decimal(
+        description=(
+            "Deprecated: use priceAmount instead.\n"
```

Comment:
Can't we keep using the `price` field here? I mean it seems the old name seems more user-friendly and it's still the same type `Decimal` with the new one. Or am I missing something?

Reply:
I think it's fine to keep it as `price: decimal` then. Would be great if we could have a `to_field` attribute in the `Scalar` object to have no need for custom logic.